twentytwentyone-blocks/assets/css/blocks.css

```diff
@@ -59,11 +59,23 @@
 # Image
 --------------------------------------------------------------*/
 
-.wp-block-image img,
 .wp-block-image img {
 	height: auto;
 }
 
+.wp-block-image figcaption{
```

Comment:
We _could_ add this here, but I wonder if a better route would be to open a Gutenberg issue to add caption styles to `theme.json`. I can see this being something we'd want to customize in other blocks too.

Reply:
Connected to https://github.com/WordPress/gutenberg/issues/12997

Reply:
Mmmh, I see. How much should be managed by the json? Looking at these styles I'm inclined to say everything but the margins. But that's maybe because I'm not too clear on how `theme.json` should handle these spacing variables that  we use for our themes.

Reply:
I think the margins could be part of the block settings too, using custom spacing.

Reply:
I think the margins may end up being handled by JSON too eventually.